lib/WUI/netdev.c

```diff
@@ -252,6 +161,7 @@ uint32_t netdev_set_active_id(uint32_t netdev_id) {
 }
 
 bool netdev_get_current_ipv4(uint8_t *dest) {
+    // TODO: Unused ???
```

Comment:
You mean the whole function? This was something Joshy asked for (and haven't used yet). But I suspect that'll get pruned once/if we do some kind of network-setup clean up.

Reply:
Yes, whole function. Thanks for clarification. TODO removed.